budgetportal/models/pages.py

```diff
@@ -0,0 +1,145 @@
+from blocks import SectionBlock
+from budgetportal import nav_bar
+from collections import OrderedDict
+from django.core.exceptions import ValidationError
+from django.db import models
+from wagtail.admin.edit_handlers import FieldPanel, StreamFieldPanel
+from wagtail.core import blocks as wagtail_blocks
+from wagtail.core.fields import RichTextField, StreamField
+from wagtail.core.models import Page
+from wagtail.contrib.routable_page.models import RoutablePage, route
+
+
+from .gov_structure import FinancialYear
+
+
+class WagtailHomePage(RoutablePage):
+    max_count = 1
+
+
+class LearningIndexPage(RoutablePage):
+    max_count = 1
+
+
+class PostIndexPage(RoutablePage):
+    max_count = 1
+    parent_page_types = []
+    intro = RichTextField(blank=True)
+
+    content_panels = Page.content_panels + [FieldPanel("intro", classname="full")]
+
+    @route(r"^posts/")
+    def main(self, request):
+        from django.shortcuts import render
+
+        return render(request, self.get_template(request), self.get_context(request))
```

Comment:
I don't think you should have this

Reply:
It was showed in one of tutorials I watched, trying all of those things (including printing inside)